pkgs/development/libraries/tbb/default.nix

```diff
@@ -1,5 +1,8 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, coreutils }:
```

Comment:
I got an error from `cp`, but I'll run through it again to double check and pull this out if I can.